build.gradle

```diff
@@ -41,6 +41,7 @@ dependencies {
     implementation 'org.jenkins-ci.main:jenkins-core:2.60.1'
     implementation 'org.jenkins-ci.plugins:credentials:2.1.10@jar'
     implementation 'org.jenkins-ci.plugins:plain-credentials:1.0@jar'
+    implementation 'org.antlr:antlr4-runtime:4.8'
```

Comment:
Required to create an instance of SynopsysCredentialsHelper

Reply:
That is absolutely correct and absolutely a bug in the current implementation of the library.

Both the Credentials plugin and the plain-credentials plugin are plugin dependencies of Jenkins Common. Because they are plugin dependencies, we specify @ jar to indicate that we are depending on the jarfile, not the hpi (we cannot resolve the java classes of the plugin via the hpi)

HOWEVER, @ jar pulls double duty because it also explicitly does not pull down an artifact's transitive dependencies. Antlr is one such transitive.

In this case, instead of depending on the individual transitives of the plugins (in this case, antlr4-runtime), we should have two dependency lines for any of our plugin dependencies-- one which grabs the jar, and the other which will grab the hpi and transitive dependencies.

Like so:

implementation 'org.jenkins-ci.plugins:credentials:2.1.10@jar'
implementation 'org.jenkins-ci.plugins:credentials:2.1.10'
implementation 'org.jenkins-ci.plugins:plain-credentials:1.0@jar'
implementation 'org.jenkins-ci.plugins:plain-credentials:1.0'